test/EntityFramework6.Npgsql.Tests/EntityFrameworkMigrationTests.cs

```diff
@@ -524,6 +524,20 @@ public void MoveTableOperationNewSchemaIsNull()
         }
 
         [Test]
+        public void MoveTableOperationPrequotedSchema()
+        {
+            var operations = new List<MigrationOperation>();
+            operations.Add(new MoveTableOperation("someOldSchema.someTable", "\"prequotedNewSchema\""));
```

Comment:
What about ```"someOldSchema"."someTable"```. Maybe another case for this would be nice. It would exercise the code path that parses the identifier into schema + table name.